zcash_proofs/src/lib.rs

```diff
@@ -19,7 +19,7 @@ use directories::BaseDirs;
 use std::path::PathBuf;
 
 pub mod circuit;
-mod constants;
+pub mod constants;
```

Comment:
The `zcash_proofs::constants` module contains various constants that we haven't yet decided on exposing. I would prefer that we only expose in this PR the components needed for enabling use of the fixed generators:

- The `FixedGeneratorOwned` type alias.
- `generate_circuit_generator`
- The six Zcash fixed generators:
  - `PROOF_GENERATION_KEY_GENERATOR`
  - `NOTE_COMMITMENT_RANDOMNESS_GENERATOR`
  - `NULLIFIER_POSITION_GENERATOR`
  - `VALUE_COMMITMENT_VALUE_GENERATOR`
  - `VALUE_COMMITMENT_RANDOMNESS_GENERATOR`
  - `SPENDING_KEY_GENERATOR`

Please change all other `pub` types in `zcash_proofs::constants` to `pub(crate)`.

Reply:
Very good, thanks. Updated the PR.